src/discordbot.ts

```diff
@@ -89,10 +89,7 @@ export class DiscordBot {
     let chan;
     let embed;
     const mxClient = this.bridge.getClientFactory().getClientAs();
-    return this.LookupRoom(guildId, channelId).then((channel) => {
```

Comment:
I think this diff is actually in the wrong spot now that I've looked a bit closer at it. This is on the matrix -> discord side, not the discord -> matrix side (the flow causing issues).